yrs/benches/benches.rs

```diff
@@ -432,6 +432,28 @@ fn b4(c: &mut Criterion, name: &str) {
     );
 }
 
+fn b4_2(c: &mut Criterion, name: &str) {
```

Comment:
I've added a benchmark that's used solely for checking out how long will it take to apply update generated from B4.1 (real live edit trace). Apparently applying update that's almost 400kB takes ~2-3ms.